pulser-simulation/pulser_simulation/hamiltonian.py

```diff
@@ -307,10 +310,9 @@ def _update_noise(self) -> None:
             )
             self._bad_atoms = dict(zip(self._qid_index, dist))
         if "doppler" in self.config.noise_types:
+            temp = cast(float, self.config.temperature) * 1e-6
```

Comment:
That can be solved by allowing the temperature to be zero (np.random.normal allows 0 value -> returns an array with zeros)

Reply:
I agree with you but I'd say that if the temperature is 0, `"doppler"` should not even be included in the noise types.

Reply:
Yep, I think so as well :) You could have a matching "doppler" not activated in the noise types <-> temperature is zero (in my opinion, it would be more speaking to see that temperature is zero when the user did not define "doppler" in noise types rather than seeing None)

Reply:
Yes, that's how it is now after the latest changes